src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html * {
```

Comment:
Don't use * selector (it impacts performance)
Variables you can specify for the `:root` selector https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties. And font to `html` element

Reply:
Thanks for the review, I got it! 
I've tried to add font-styles only to `html` element but _user agent stylesheet_ overrides it every time. Otherwise, I needed to write a  font-style for each `form` element separately.